client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -749,17 +749,31 @@ public void testUpdateAliases() throws Exception {
         {
             // tag::update-aliases-request
```

Comment:
I'm curious why the sections were all called "update-aliases-*" before. The request name (IndicesAliasesRequest) doesn't seem to have changed recently. Maybe it would make sense to update the tags (and the corresponding includes) as well now?

Reply:
this is on me, it's a common inconsistency between our Client method names, our request names, and how the API are called in our SPEC. We are now following the name from the SPEC when naming the new methods. But we leave requests unchanged to reduce noise and not impact bw comp too much at this stage. Let's leave the tags as they are?